puma/plot_base.py

```diff
@@ -3,18 +3,19 @@
 from dataclasses import dataclass
 
 import atlasify
-from matplotlib import gridspec
+from matplotlib import axis, gridspec
 from matplotlib.figure import Figure
 
-from puma.utils import logger, set_xaxis_ticklabels_invisible
+from puma.utils import set_xaxis_ticklabels_invisible
+from puma.utils.logging import logger
 
 atlasify.LINE_SPACING = 1.3  # overwrite the default, which is 1.2
 
 
 # TODO: enable `kw_only` when switching to Python 3.10
 # @dataclass(kw_only=True)
 @dataclass
-class PlotLineObject:
+class plot_line_object:
```

Comment:
let's here directly keep the new naming, also below

Reply:
yeah I changed it back for `PlotLineObject` and `PlotObject`.

Do you think we should also already update `histogram_plot`, `histogram`, `roc_plot`, ... or should we move this in another PR?

Reply:
we can move this to another MR